50_HP1000.pm

```diff
@@ -1324,6 +1328,7 @@ sub HP1000_PushWU($$) {
     my $wu_user         = AttrVal( $name, "wu_id", "" );
     my $wu_pass         = AttrVal( $name, "wu_password", "" );
     my $wu_realtime     = AttrVal( $name, "wu_realtime", undef );
+    my $wu_pushURL      = AttrVal( $name, "wu_pushURL", undef );
```

Comment:
There is extSrvPush_Url already, pls remove this from the patch to avoid duplication of the same functionality.